turbo.json

```diff
@@ -10,51 +10,57 @@
         "generated/spec.schema.json"
       ]
     },
-    "generate:typescript": {
+    "@realm/bindgen#build": {
       "inputs": [
-        "src/*",
-        "spec.yml"
+        "src/*"
+      ],
+      "dependsOn": [
+        "generate:spec-schema"
+      ]
+    },
+    "@realm/binding#compile": {
+      "inputs": [
+        "generated/node/*"
       ],
       "outputs": [
-        "generated/ts/*"
+        "build/Release/*",
+        "build/Debug/*"
       ],
       "dependsOn": [
-        "generate:spec-schema"
+        "^build"
       ]
     },
-    "generate:node": {
+    "@realm/binding#build": {
       "inputs": [
-        "src/*",
-        "spec.yml"
+        "src/*"
       ],
       "outputs": [
-        "generated/node/*"
+        "dist/*"
       ],
       "dependsOn": [
-        "generate:spec-schema"
+        "compile"
       ]
     },
-    "compile:node": {
+    "realm#test:types": {
       "inputs": [
-        "generated/node/*"
+        "type-tests/*"
       ],
+      "outputs": [],
       "dependsOn": [
-        "generate:node"
+        "build"
       ]
     },
-    "compile": {
-      "inputs": [],
+    "realm#test": {
       "dependsOn": [
-        "compile:node"
+        "test:types"
```

Comment:
I don't think you want this here since it reduces parallelism. The type tests can be validated while this runs.

Reply:
I don't see any good alternative that allows me to run both when calling `npx turbo run test`.

Reply:
In a traditional build system you would do something like having "test" be an alias/phony/dummy/no-op target that just depends on whatever tasks you actually want to run when you type "test". That way you can still have a single command to run all tests, but each subtask are allowed to happen in parallel as soon as its actual requirements are met.

Interestingly, the type-tests don't depend on the compiled module, just the generated .d.ts files, so they could run much earlier than the executing tests (not that we currently expose enough granularity to turbo to take advantage of that yet, but we should at some point)